docs/source/conf.py

```diff
@@ -261,6 +263,8 @@ class AccessorMethodDocumenter(AccessorLevelDocumenter, MethodDocumenter):
 
 
 def setup(app):
+    os.makedirs("/home/docs/.ipython/profile_default/startup")
+    shutil.copy("disable-warnings.py", "/home/docs/.ipython/profile_default/startup/")
```

Comment:
They let you run arbitrary code on start-up? 😨

Reply:
Yep, apparently! This is what the support team suggested

Reply:
Let the evil commence![](https://media3.giphy.com/media/wlBS2Aif8eBvW/giphy.gif)